### PR TITLE
22130 integrator api steps linkage

### PR DESCRIPTION
**Pull request by nnorbert**
Journey step connections.

### CODE REVIEW

File: apps/integrator/app/types/index.ts [hunk 1]
```diff
@@ -47,3 +40,22 @@ export type Submission = {
...
+  destinationStepId: string;
+};
+
+export type Journey = {
+  id: string;
+  title: string;
+  userId: string;
+  organizationId: string;
+  status: string;
+  initialStepId: string;
```
Comment (peschina) [inline apps/integrator/app/types/index.ts:56]: shouldn't `initialStepId` be optional?

**Review by peschina — APPROVED** [review]
Looks good, just one comment

File: apps/integrator/app/types/index.ts (hunk 1 above)
Comment (nnorbert) [reply apps/integrator/app/types/index.ts:56]: Yes, it should, good catch! Thank you!

Comment (nnorbert) [reply apps/integrator/app/types/index.ts]: Done

**Review by peschina — APPROVED** [review]

**Review by alfonsograziano — APPROVED** [review]
LGTM!

**Pull request merged**